stories/bannertest.stories.mdx

```diff
@@ -0,0 +1,39 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs/blocks";
+import Banner from "../src/lib/components/banner/Banner.component";
+
+<Meta title="MDX/Banner" component={Banner} />
+
+# Banner
+
+With `MDX` we can define a story for `Banner` right in the middle of our
+Markdown documentation.
+
+<Canvas>
+  <Story name="danger">
+    <Wrapper>
+      <Banner
+        variant="danger"
+        icon={<i className="fas fa-exclamation-triangle" />}
+        title={"Error"}
+      >
+        {"There is an error."}
+      </Banner>
+    </Wrapper>
+  </Story>
+</Canvas>
```

Comment:
Discussion more than something related to the implementation itself : it could be interresting to be able to have a live playground in the documentation to see the impacts of the component when playing with a property. MDX support this thanks to a special code annotation  (https://mdxjs.com/guides/live-code)
``` 
```js live=true ```
```

Not sure through if this is supported out of the box by storybook but it maybe worth the try